core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -138,9 +142,29 @@ static ClientRequestContext of(RpcRequest request, URI uri) {
     @Override
     ClientRequestContext newDerivedContext(Request request);
 
+    /**
+     * Creates a new derived {@link RequestContext} with the specified {@link Request} and {@link Endpoint}
+     * which the {@link RequestLog} is different from the deriving context.
```

Comment:
`with a different {@link RequestLog} than this one.`

Reply:
Rewrote the method Javadoc. Please review again.